lib/swagger-ui/swagger-ui.ts

```diff
@@ -0,0 +1,72 @@
+import * as swaggerUi from 'swagger-ui-dist';
+import { favIconHtml, htmlTemplateString, jsTemplateString } from './constants';
+import { SwaggerCustomOptions } from '../interfaces';
```

Comment:
This file could use some refactoring:

- consts to functions
- add missing TS types
- make variables more descriptive/add comments in "buildJSInitOptions"